apim-lab/4-policyExpressions/apimanagement-4-3-Transformation.md

```diff
@@ -151,7 +151,7 @@ Query string parameters and headers can be easily modified prior to sending the
   </inbound>
   ```
 
-- Test the call by using either the *Starter* or *Unlimited* product, then inspect the result on the *Trace* tab.
+- Test the call by using either the *Starter* or *Unlimited* product, click on Trace button (if Tracing is not enabled, Enable Tracing) and then inspect the result on the *Trace* tab.

```

Comment:
Please consider changing to something like this: 

click on the Trace button, and then inspect the results on the *Trace* tab. If tracing is not enabled, press *Enable tracing*.